examples/step-64/step-64.cc

```diff
@@ -633,15 +637,6 @@ int main(int argc, char *argv[])
 
       Utilities::MPI::MPI_InitFinalize mpi_init(argc, argv, 1);
 
-      int         n_devices       = 0;
-      cudaError_t cuda_error_code = cudaGetDeviceCount(&n_devices);
-      AssertCuda(cuda_error_code);
-      const unsigned int my_mpi_id =
-        Utilities::MPI::this_mpi_process(MPI_COMM_WORLD);
-      const int device_id = my_mpi_id % n_devices;
-      cuda_error_code     = cudaSetDevice(device_id);
-      AssertCuda(cuda_error_code);
-
```

Comment:
Should we add a comment that people that people need to use `--kokkos-map_device_id_by=mpi_rank` if they have multiple GPUs per node?

Reply:
That's the default anyway, right?

Reply:
You are right. We actually switched to use GPU 0 by default but then changed back the behavior